docs/ref/models/fields.txt

```diff
@@ -1157,6 +1157,35 @@ you can use the name of the model, rather than the model object itself::
         # ...
         pass
 
+Note that relationships defined this way on
+:ref:`abstract models <abstract-base-classes>` are only resolved when the
+model is subclassed as concrete model and are not relative to the abstract
+model's ``app_label``::
+
+    from django.db import models
+
+    class AbstractCar(models.Model):
+        manufacturer = models.ForeignKey(
+            'Manufacturer',
+            on_delete=models.CASCADE,
+        )
+
+        class Meta:
+            app_label = 'products'
+            abstract = True
+
+    class Manufacturer(models.Model):
+        # ...
+
+        class Meta:
+            app_label = 'production'
+
+    class Car(AbstractCar):
+        class Meta:
+            app_label = 'production'
+
+        # Car.manufacturer will point to `production.Manufacturer` here.
```

Comment:
@AlexHill Good idea, I'll create a new ticket to track this documentation update, feel free to assign it to yourself.

It would be great if it also included a note about using _relative_ references to define reusable blueprint models.